README.md

```diff
@@ -460,6 +461,19 @@ To prevent data loss, this unsafe task execution must be confirmed by parameter
 
 Replicate installed CRX package to other AEM instance(s).
 
+#### Task 'aemDownload'
+
+Builds and downloads CRX package from AEM instance. Similar to [aemCheckout](#task-aemcheckout) but using CRX package as an export method.
```

Comment:
Builds and downloads CRX package from AEM instance. Similar to [aemCheckout](#task-aemcheckout) but produces CRX package that could be optionally automatically extracted.